src/current/v25.2/jsonpath.md

```diff
@@ -0,0 +1,567 @@
+---
+title: JSONPath Queries
+summary: Use JSONPath expressions, functions, operators, and methods to navigate, extract, and transform JSONB data.
+toc: true
+docs_area: reference.sql
+---
+
+{% include_cached new-in.html version="v25.2" %} JSONPath expressions and functions are used to query and filter [`JSONB`]({% link {{ page.version.version }}/jsonb.md %}) data. A [JSONPath expression](#jsonpath-expression) is a string that identifies one or more elements in a JSON document, and is used as a [JSONPath function](#jsonpath-functions) argument.
+
+## JSONPath expression
+
+A JSONPath expression has the following generic structure:
+
+~~~
+[mode] $[.key] [? (filter_exp)].[key].[method]
+~~~
+
+- `mode` is an optional mode (`lax` or `strict`) that determines how structural mismatches are tolerated. If not specified, the mode is `lax`. Refer to [Structural error handling](#structural-error-handling).
+- `$` is the root of the JSONPath, and must be the first element in a JSONPath expression. For an example, refer to [Access entire document](#access-entire-document).
+- `.key` is an optional key accessor that refers to a named field in a `JSONB` object. To access array elements, use `.key[index]` or `.key[*]`. Each successive key accessor navigates one level deeper into the structure. `.*` matches all fields in the current object.
```

Comment:
We should make mention of the array accessor as it's own thing, I think. For example, you can do this:
```
root@localhost:26257/defaultdb> SELECT jsonb_path_query('[1, 2, 3]', '$[0]');
  jsonb_path_query
--------------------
                 1
(1 row)
```